posts/2023-02-28-instant-on-beta-update.adoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Liberty InstantOn beta update"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/tjwatson
+author_github: https://github.com/tjwatson
+seo-title: Liberty InstantOn beta update - OpenLiberty.io
+seo-description: The Open Liberty 23.0.0.2-beta brings new enhancements to the InstantOn feature that make it easier to build and deploy Jakarta EE and MicroProfile applications with incredibly fast startup times.
+blog_description: "The Open Liberty 23.0.0.2-beta brings new enhancements to the InstantOn feature that make it easier to build and deploy Jakarta EE and MicroProfile applications with incredibly fast startup times."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Liberty InstantOn beta update
+Thomas Watson <https://github.com/tjwatson>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+The Open Liberty 22.0.0.11-beta introduced InstantOn, an exciting new feature that provides incredibly fast startup times for MicroProfile and Jakarta EE applications. Since the initial InstantOn beta there have been a few changes and enhancements to Liberty InstantOn which make it easier to create and deploy applications with Liberty InstantOn. The remainder of this article uses the project setup of the original InstantOn article for reference.  If you would like to follow along then you should read and follow the instructions in the link:https://openliberty.io/blog/2022/09/29/instant-on-beta.html[Liberty InstantOn startup for cloud native Java applications] article first.
```

Comment:
```suggestion
The Open Liberty 22.0.0.11-beta introduced InstantOn, an exciting new feature that provides incredibly fast startup times for MicroProfile and Jakarta EE applications. Since this initial beta release, we've made a few changes and enhancements to make it easier to create and deploy applications with Liberty InstantOn. 

This post uses the same project setup from our original InstantOn article, link:https://openliberty.io/blog/2022/09/29/instant-on-beta.html[Liberty InstantOn startup for cloud native Java applications].  If you would like to follow along with the examples here, read and follow the instructions in that post first.

The following sections describe the improvements to Liberty InstantOn that are included with the 23.0.0.2 beta release.
```